### PR TITLE
Add localization messages for 'cs' locale

### PR DESCRIPTION
**Pull request by Laupe**
add Czech Localization

### CODE REVIEW

File: js/localization/cs.json [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        "dxDataGrid-filterRowOperationEquals": "Rovná se",
+        "dxDataGrid-filterRowOperationNotEquals": "Nerovná se",
+        "dxDataGrid-filterRowOperationLess": "Menší",
+        "dxDataGrid-filterRowOperationLessOrEquals": "Menší nebo rovno",
+        "dxDataGrid-filterRowOperationGreater": "Větší",
+        "dxDataGrid-filterRowOperationGreaterOrEquals": "Větší nebo rovno",
+        "dxDataGrid-filterRowOperationStartsWith": "Žačíná na",
+        "dxDataGrid-filterRowOperationContains": "Obsahuje",
+        "dxDataGrid-filterRowOperationNotContains": "Neobsahuje",
+        "dxDataGrid-filterRowOperationEndsWith": "Termine avec",
```
Comment (BingoRUS) [inline js/localization/cs.json]: Hi, @Laupe!
"Termine avec" - is it the right translation? It looks like French :)

File: js/localization/cs.json [hunk 2]
```diff
@@ -0,0 +1,239 @@
+{
+    "cs": {
+        "Yes": "Ano",
+        "No": "Ne",
+        "Cancel": "Zrušit",
+        "Clear": "Smazat",
+        "Done": "Ok",
```
Comment (BingoRUS) [inline js/localization/cs.json]: Hi, @Laupe!
Is it right to use "Ok"? It might be confused with another OK constant string here. How about "Hotový"?

Comment (jdvjdv82) [reply js/localization/cs.json]: Done could be "OK" or "Hotovo" or "Dokončit". It depends on context

File: js/localization/cs.json (hunk 1 above)
Comment (jdvjdv82) [reply js/localization/cs.json]: "Termine avec" is "Končí na".
and there is a typo in "dxDataGrid-filterRowOperationStartsWith": "Začíná na"

**Comment by BingoRUS** [comment]
@jdvjdv82 Thanks! I've updated the file. Could you please check other constants once again before I merge these changes?

File: js/localization/cs.json [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+        "dxDataGrid-filterRowOperationNotContains": "Neobsahuje",
+        "dxDataGrid-filterRowOperationEndsWith": "Končí na",
+        "dxDataGrid-filterRowOperationBetween": "Mezi",
+        "dxDataGrid-filterRowOperationBetweenStartText": "Začíná",
+        "dxDataGrid-filterRowOperationBetweenEndText": "Končí",
+        "dxDataGrid-applyFilterText": "Použít filtr",
+        "dxDataGrid-trueText": "Platí",
+        "dxDataGrid-falseText": "Neplatí",
+        "dxDataGrid-sortingAscendingText": "Srovnat vzestupně",
+        "dxDataGrid-sortingDescendingText": "Srovnat zestupně",
```
Comment (jdvjdv82) [inline js/localization/cs.json]: "Srovnat sestupně"

File: js/localization/cs.json [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+        "dxPivotGrid-filterFields": "Filtrovat pole",
+        "dxPivotGrid-allFields": "Všechna pole",
+        "dxPivotGrid-columnFieldArea": "Zde vložte pole sloupců",
+        "dxPivotGrid-dataFieldArea": "Zde vložte pole dat",
+        "dxPivotGrid-rowFieldArea": "Zde vložte pole řádků",
+        "dxPivotGrid-filterFieldArea": "Zde vložte filtr pole",
+
+        "dxScheduler-editorLabelTitle": "Předmět",
+        "dxScheduler-editorLabelStartDate": "Počáteční datum",
+        "dxScheduler-editorLabelEndDate": "Konecné datum",
```
Comment (jdvjdv82) [inline js/localization/cs.json]: Koncové datum

Comment (jdvjdv82) [reply js/localization/cs.json]: @BingoRUS the rest looks OK. So i already finished the review. Only those 2 changes are needed. BTW we don't have the term for "tree list" i believe

Comment (BingoRUS) [reply js/localization/cs.json]: @jdvjdv82 Thanks, I applied your suggestions and going to merge Czech localization now.

**Review by BingoRUS — APPROVED** [review]

**Pull request merged**